pkg/runtime/pubsub/subscriptions_test.go

```diff
@@ -798,3 +798,26 @@ func TestK8sSubscriptions(t *testing.T) {
 		assert.Equal(t, "testValue", subs[0].Metadata["testName"])
 	}
 }
+
+func TestGetRuleMatchString(t *testing.T) {
+	var cases = []subscriptionsapiV2alpha1.Rule{
```

Comment:
The linter wants this

```suggestion
	cases := []subscriptionsapiV2alpha1.Rule{
```

Reply:
Done.